src/content/docs/configuration/providers.mdx

```diff
@@ -251,33 +145,193 @@ When following the [Set a Target procedure](#set-a-target), note the following c
 2. **Remote Hostname**
 
     The hostname of the remote host.
+
 3. **Remote Password**
 
     The password for the remote user.
     If password authentication is disabled on the remote host, you can choose an SSH key in the next step.
+
 4. **Remote Port**
 
     The TCP port used to access the remote host's SSH daemon.
     Unless otherwise configured on your remote host, this should be set to the default value (`22`)
+
 5. **Remote User**
 
      The username used to execute the required commands on the remote host.
 
+    <br />
+
     <Aside type="caution">
       Non-root user is required so correct ownership is assumed inside the
       project container.
     </Aside>
 
+    <br />
+
 6. **Sock Path**
 
     The Docker UNIX socket location on the remote host.
-    Unless otherwise configured after installing Docker, this should be set to the default value (`/var/run/docker.sock`)
+    Unless otherwise configured after installing Docker, this should be set to the default value (`/var/run/docker.sock`).
+
 7. **Remote Private Key Path**
 
     The SSH private key used to connect to the remote host.
     If you are using password authentication, you can choose "None".
 
-##### Local
+#### Local
 
 By default, Daytona comes pre-configured with a local Docker Target.
-If you have removed the default Docker Target, you can recreate it using the [Set a Target procedure](#set-a-target).
+If you have removed the default Docker Target, you can recreate it using the [set a Target](/configuration/targets#set-a-target) procedure.
+
+## AWS
```

Comment:
@stefanicjuraj seems we got our wires crossed. This part for AWS is correct but it should be in the Target section.

Here it should be step by step how to install AWS provider and configure it, that part is missing. Please reach out to @Tpuljak for more info and a account for all providers if needed

Reply:
@stefanicjuraj edit apologies configuring is target